implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMeterRegistry.java

```diff
@@ -140,15 +131,10 @@ public void stop() {
 
     @Override
     protected void publish() {
-        if (isDelta()) {
```

Comment:
Isn't this a breaking change?

Reply:
Nope. This is being moved to the OTLPMetricConverter([see here](https://github.com/micrometer-metrics/micrometer/blob/56b3607a68840fad0f6f3d5287f3f34cca8521a4/implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMetricConverter.java#L67)).